EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs

```diff
@@ -0,0 +1,93 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageWaitersPage : BasePage
+    {
+        public ManageWaitersPage(IWebDriver driver) : base(driver)
+        {
+
+        }
+
+        [FindsBy(How = How.XPath, Using = "(//button[contains(@class, 'MuiButtonBase-root')])[2]")]
+        private IWebElement _deleteButton;
+
+        [FindsBy(How = How.XPath, Using = "//button[@title='Add Waiter']")]
+        private IWebElement _addWaiterButton;
+
+        [FindsBy(How = How.XPath, Using = "//input[@name='name']")]
+        private IWebElement _inputName;
+
+        [FindsBy(How = How.XPath, Using = "//input[@name='email']")]
+        private IWebElement _inputEmail;
+
+        [FindsBy(How = How.XPath, Using = "//input[@name='password']")]
+        private IWebElement _inputPassword;
+
+        [FindsBy(How = How.XPath, Using = "//input[@name='phone_number']")]
+        private IWebElement _inputPhoneNumber;
+
+        [FindsBy(How = How.XPath, Using = "//span[text()='Add']/parent::button")]
+        private IWebElement _addButton;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Name is required']")]
+        private IWebElement _inputNameValidationWarning;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Mail is required']")]
+        private IWebElement _inputEmailValidationWarning;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Password is required']")]
+        private IWebElement _inputPasswordValidationWarning;
+
+        [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
+        private IWebElement _inputPhoneNumberValidationWarning;
+
+        public void ClickDeleteButton()
+        {
+            _deleteButton.Click();
+        }
+        public void ClickAddWaiterButton()
```

Comment:
please add one empty line between the methods so that it would be better readability